contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/test_import_order.py

```diff
@@ -135,9 +135,9 @@ def test_multiple_imports_error(self):
     self.assertEqual(1, len(import_chunks))
     module_types, chunk_errors = io.classify_imports(import_chunks[0])
     self.assertEqual(3, len(chunk_errors))
-    self.assertItemsEqual(['T403', 'T405', 'T402'],
-                          [chunk_error.code for chunk_error in chunk_errors])
-    self.assertItemsEqual([ImportType.STDLIB, ImportType.THIRD_PARTY], module_types)
+    self.assertEqual(sorted(['T403', 'T405', 'T402']),
```

Comment:
`assertItemsEqual` was renamed in Python3 to  `assertCountEqual`. Calling `sorted` has the same effect, and is more explicit and cleaner than choosing the test function according to interpreter version.

Refer to https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertItemsEqual